### PR TITLE
Compile testing for StorIOContentResilverAnnotationsProccesor

### PR DESCRIPTION
**Pull request by geralt-encore**

### CODE REVIEW

**Comment by nikitin-da** [comment]
Great work man! 👍

**Comment by nikitin-da** [comment]
@artem-zinnatullin would you like to read this request? Or maybe we can merge it, because it is similar to sqlite part in many respects.

File: storio-content-resolver-annotations-processor-test/build.gradle [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    defaultConfig {
+        minSdkVersion rootProject.ext.minSdkVersion
+    }
+}
+
+dependencies {
+    testCompile libraries.storIOContentResolver
+    testCompile libraries.storIOContentResolverAnnotationsProcessor
+    testCompile libraries.googleTestingCompile
+    testCompile files(getRuntimeJar())
```
Comment (geralt-encore) [inline storio-content-resolver-annotations-processor-test/build.gradle]: Dunno it's not really a library. I mean it doesn't have versioning so moving it to libraries won't give much profit I guess.

Comment (artem-zinnatullin) [inline storio-content-resolver-annotations-processor-test/build.gradle]: well, technically it's a library, just a jar on your system. Point is to remove code duplication from build gradle, up2u though :)

Comment (geralt-encore) [inline storio-content-resolver-annotations-processor-test/build.gradle]: @artem-zinnatullin done :)

**Comment by geralt-encore** [comment]
Ouch, I forgot to squash a PR 😔

**Comment by artem-zinnatullin** [comment]
That is my fault, don't blame yourself, I'll enable squash-only option in repo settings!

**Comment by geralt-encore** [comment]
I just got used to it as default option so didn't even check. But it's nice that that you enabled it now.

**Comment by artem-zinnatullin** [comment]
Yeah, I know that feel bro, a month or so ago I've also merged PR in some of my repos without squashing because I got used to it as default option…

**Pull request merged**